wpilibc/src/main/native/cpp/InterruptableSensorBase.cpp

```diff
@@ -14,6 +14,32 @@
 
 using namespace frc;
 
+InterruptableSensorBase::~InterruptableSensorBase() {
+  if (m_interrupt == HAL_kInvalidHandle) return;
+  int32_t status = 0;
+  auto param = HAL_CleanInterrupts(m_interrupt, &status);
+  if (param) {
+    delete reinterpret_cast<InterruptEventHandler*>(param);
+  }
+  // Ignore status, as an invalid handle just needs to be ignored.
+  m_interrupt = HAL_kInvalidHandle;
+}
+
+InterruptableSensorBase::InterruptableSensorBase(InterruptableSensorBase&& rhs)
+    : ErrorBase(std::move(rhs)), m_interrupt(rhs.m_interrupt.load()) {
+  rhs.m_interrupt = HAL_kInvalidHandle;
+}
+
+InterruptableSensorBase& InterruptableSensorBase::operator=(
+    InterruptableSensorBase&& rhs) {
+  ErrorBase::operator=(std::move(rhs));
+
+  m_interrupt = rhs.m_interrupt.load();
+  rhs.m_interrupt = HAL_kInvalidHandle;
```

Comment:
Should this use rhs.m_interrupt.exchange()?

Reply:
As its in the move operator, likely doesn't matter, but either way fixed.